library/Icinga/File/Ini/IniParser.php

```diff
@@ -260,6 +261,12 @@ public static function parseIniFile($file)
             throw new NotReadableError('Couldn\'t read the file `%s\'', $path);
         }
 
-        return Config::fromArray(parse_ini_string($content, true))->setConfigFile($file);
+        try {
+            $configArray = parse_ini_string($content, true);
+        } catch (ErrorException $e) {
+            throw new ConfigurationError('Couldn\'t parse the INI file `%s\'', $path, $e);
```

Comment:
Please include the error message in the output, as previously suggested. Other code parts also use `$e->getMessage()`.

Reply:
"Other code parts" are hiding the previous exception + stack trace.
`IcingaException` was designed to do that better – as I did.

Reply:
Ok, let's skip the discussion on other code parts.

Your current change doesn't show the error message, which would help find the problem more easily.

![screen shot 2018-01-19 at 10 23 58](https://user-images.githubusercontent.com/382049/35143806-ead3b75a-fd02-11e7-9da0-85060b9e375c.png)

Reply:
So the previous exception (message) is not shown? Strange. I'll have a look at this...

Reply:
Awesome, thanks ❤️